### PR TITLE
[ᚬrc/v0.14.0] feat: Tweak consensus parameters

### PR DESCRIPTION
**Pull request by doitian**
- Change target epoch duration to 4 hours
- Reduce epoch reward to 1/4
- Increase secondary epoch reward to 600,000 bytes

### CODE REVIEW

File: resource/specs/testnet.toml [hunk 1]
```diff
@@ -4,10 +4,11 @@ name = "ckb_testnet"
 version = 0
 parent_hash = "0x0000000000000000000000000000000000000000000000000000000000000000"
 timestamp = 1558137600000
-difficulty = "0x1000"
+# difficulty of the block 135599 in rylai-v2
+difficulty = "0x5a7b"
```
Comment (doitian) [inline resource/specs/testnet.toml]: This will change to the last epoch of rylai-v2 before the release deadline.

File: spec/src/consensus.rs [hunk 2]
```diff
@@ -24,10 +24,10 @@ pub(crate) const ORPHAN_RATE_TARGET_RECIP: u64 = 20;
...
 const MIN_BLOCK_INTERVAL: u64 = 8 * 1000; // 8s
 const TWO_IN_TWO_OUT_CYCLES: Cycle = 2_580_000;
-pub(crate) const EPOCH_DURATION_TARGET: u64 = 8 * 60 * 60 * 1000; // 8 hours
-pub(crate) const MAX_EPOCH_LENGTH: u64 = EPOCH_DURATION_TARGET / MIN_BLOCK_INTERVAL; // 3600
-pub(crate) const MIN_EPOCH_LENGTH: u64 = EPOCH_DURATION_TARGET / MAX_BLOCK_INTERVAL; // 480
-pub(crate) const GENESIS_EPOCH_LENGTH: u64 = 2_000;
+pub(crate) const EPOCH_DURATION_TARGET: u64 = 4 * 60 * 60 * 1000; // 4 hours
+pub(crate) const MAX_EPOCH_LENGTH: u64 = EPOCH_DURATION_TARGET / MIN_BLOCK_INTERVAL; // 1800
+pub(crate) const MIN_EPOCH_LENGTH: u64 = EPOCH_DURATION_TARGET / MAX_BLOCK_INTERVAL; // 240
+pub(crate) const GENESIS_EPOCH_LENGTH: u64 = MAX_EPOCH_LENGTH;
```
Comment (doitian) [inline spec/src/consensus.rs]: @zhangsoledad @nirenzang not sure how to set the genesis epoch length. The old value 2000 is out of range, so I just set it to `MAX_EPOCH_LENGTH`.

**Comment by zhangsoledad** [comment]
/home/soledad/project/ckb/chain.rsL755 
`consensus.epoch_duration_target = 8 * 60 * 60 * 1000;`
fix test

**Comment by doitian** [comment]
I'll update difficulty in another PR, so this will not be blocked.

**Pull request merged**